### ISSUE TITLE
Error Thrown by DataTableAbstract::getColumnNameByIndex()  on Laravel 10

### ISSUE THREAD

**Issue by fahmidyt** [yajra/laravel-datatables#3010]
### Summary of problem or feature request

When I try to update Laravel 9 -> 10. There is a bug shown. And it is caused when the method `->toJson` is called. The reason behind that is that I use the `DB::raw()` on my Query Builder and it caused the `DataTableAbstract::getColumnNameByIndex()` thrown error. The error was basically caused by type error. Details on the following snippets.

### Code snippet of problem

The following snippets are from my controller.
```php
class Controller extends MYController {
    public function read(Request $request) {
        $queryBuilder = DB::table('tbl_association')->select('id', 'association_name');
        // this line will cause the bug
        $queryBuilder->addSelect(DB::raw('(SELECT 1)'));

        return DataTables::query($queryBuilder)->toJson();
    }
}
```

And the following snippets are the error has thrown
```json
{
    "message": "Yajra\\DataTables\\DataTableAbstract::getColumnNameByIndex(): Return value must be of type string, Illuminate\\Database\\Query\\Expression returned",
    "exception": "TypeError",
    "file": "C:\\folder\\to\\project\\vendor\\yajra\\laravel-datatables-oracle\\src\\DataTableAbstract.php",
    "line": 1030,
    "trace": ["..."]
}
```

And I also read this article [https://laravel.com/docs/10.x/upgrade#database-expressions](https://laravel.com/docs/10.x/upgrade#database-expressions)
There's a thing that might be related to this

```
Notably, the grammar's raw string value must now be retrieved via the expression's getValue(Grammar $grammar) method. Casting an expression to a string using (string) is no longer supported.
```

And I hope you can help with this.. Thanks!

### System details

- Operating System: Windows 11
- PHP Version: 8.1.10
- Laravel Version: 10.13.2
- Laravel-Datatables Version: 10.1.0

**Comment by Nacoma** [comment]
Still an ongoing issue.

**Comment by yajra** [comment]
Fixed vi #3029, v10.6.2. Thanks!
